mappings/net/minecraft/loot/entry/TagEntry.mapping

```diff
@@ -1,6 +1,13 @@
 CLASS net/minecraft/class_91 net/minecraft/loot/entry/TagEntry
-	FIELD field_1005 name Lnet/minecraft/class_3494;
+	FIELD field_1005 tag Lnet/minecraft/class_3494;
```

Comment:
This should still be `name` - that's the property in the JSON format

Reply:
```suggestion
	FIELD field_1005 name Lnet/minecraft/class_3494;
```